client/my-sites/upgrades/checkout/checkout.jsx

```diff
@@ -306,12 +310,18 @@ const Checkout = React.createClass( {
 	}
 } );
 
+function mapStateToProps ( state ) {
+	const cards = getStoredCards( state );
+	const selectedSite = getSelectedSite( state );
```

Comment:
Another thing is that `selectedSite` from global state is not always guaranteed to exist. ( For example an empty cache on a first page load ). It looks like we didn't handle it before, but we'll want to check if the site exists before accessing any props.

Thanks @spen! This is pretty close, but we'll want to bullet proof this case.